src/main/webapp/jsp/instructorFeedbackEdit.jsp

```diff
@@ -1,1324 +1,52 @@
-                                        </td>
-                                        <td>
-                                            <input class="visibilityCheckbox answerCheckbox<%= question.questionNumber %>"
-                                                type="checkbox" value="<%= FeedbackParticipantType.INSTRUCTORS %>" disabled="disabled"
-                                                <%= question.showResponsesTo.contains(FeedbackParticipantType.INSTRUCTORS) ? "checked=\"checked\"" : "" %>>
-                                        </td>
-                                        <td>
-                                            <input class="visibilityCheckbox giverCheckbox<%= question.questionNumber %>"
-                                                type="checkbox" value="<%= FeedbackParticipantType.INSTRUCTORS %>" disabled="disabled"
-                                                <%= question.showGiverNameTo.contains(FeedbackParticipantType.INSTRUCTORS) ? "checked=\"checked\"" : "" %>>
-                                        </td>
-                                        <td>
-                                            <input class="visibilityCheckbox recipientCheckbox<%= question.questionNumber %>"
-                                                type="checkbox" value="<%= FeedbackParticipantType.INSTRUCTORS %>" disabled="disabled"
-                                                <%= question.showRecipientNameTo.contains(FeedbackParticipantType.INSTRUCTORS) ? "checked=\"checked\"" : "" %>>
-                                        </td>
-                                    </tr>
-                                </table>
-                            </div>
-                        </div>
-                        <div>
-                            <span class="pull-right">
-                                <input id="button_question_submit-<%= question.questionNumber %>"
-                                       type="submit" class="btn btn-primary"
-                                       value="Save Changes" tabindex="0"
-                                       style="display:none">
-                            </span>
-                        </div>
-                    </div>
-                </div>
-                <input type="hidden" name="<%= Const.ParamsNames.FEEDBACK_SESSION_NAME %>" value="<%= data.session.feedbackSessionName %>">
-                <input type="hidden" name="<%= Const.ParamsNames.COURSE_ID %>" value="<%= data.session.courseId %>">
-                <input type="hidden" name="<%= Const.ParamsNames.FEEDBACK_QUESTION_ID %>" value="<%= question.getId() %>">
-                <input type="hidden" name="<%= Const.ParamsNames.FEEDBACK_QUESTION_NUMBER %>" value="<%= question.questionNumber %>">
-                <input type="hidden" name="<%= Const.ParamsNames.FEEDBACK_QUESTION_TYPE %>" value=<%= question.questionType.toString() %>>
-                <input type="hidden" name="<%= Const.P
```

Comment:
Should the additional settings tag only take in the additional settings data structure?
Or does it require other data from the overall form as well?